packages/core/src/__tests__/unit/application.unit.ts

```diff
@@ -305,15 +285,63 @@ describe('Application', () => {
       expect(binding.key).to.equal('services.my-service');
       expect(findKeysByTag(app, 'service')).to.containEql(binding.key);
     });
+  });
 
-    function givenApp() {
-      app = new Application();
+  describe('shutdown signal listener', () => {
+    it('registers a SIGTERM listener when app starts', async () => {
+      const count = getListeners().length;
```

Comment:
Any reason we aren't checking with actual values like 0 or 1?  Is the getListeners() going to return a variable amount depending on a system option or something?

Reply:
`process` is a global. We cannot assert the listener count is `0` or `1`.